bin/publish-modules

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env node
+
+var exec = require( 'child-process-promise' ).exec;
+var semver = require( 'semver' );
+
+console.log( 'Modules requiring publish:' );
```

Comment:
Does this call belong to any of the functions? It seems a little weird to have it outside of everything.